wcivf/apps/elections/templates/elections/includes/_polling_place.html

```diff
@@ -23,28 +23,34 @@ <h2 id='where'>
             </p>
         {% else %}
             {% if polling_station.custom_finder %}
-                <p>{% You can find your polling station by
-                    <a href="{{ polling_station.custom_finder }}">
-                        following this link</a>.</p>
+                {% blocktrans trimmed with custom_finder=polling_station.custom_finder %}
+                    <p>You can find your polling station by
+                        <a href="{{ custom_finder }}">
+                            following this link</a>.</p>
+                {% endblocktrans %}
             {% else %}
                 {% if ballots_today and not multiple_city_of_london_elections_today %}
-                    <strong>Polling stations are open from {{ ballots_today.0.election.polls_open|time:"ga" }} till {{ ballots_today.0.election.polls_close|time:"ga" }} today.</strong>
+                    {% blocktrans trimmed with open_time=ballots_today.0.election.polls_open|time:"ga" close_time=ballots_today.0.election.polls_close|time:"ga" %}
+                        <strong>Polling stations are open from {{ open_time }} till {{ close_time }} today.</strong>
+                    {% endblocktrans %}
                 {% endif %}
                 {% if polling_station.addresses %}
-                    <p>Your polling station in {{ postcode }} depends on your address. <a href="https://wheredoivote.co.uk/postcode/{{ postcode }}/">Check the correct polling station for your address &raquo;</a></p>
+                    {% blocktrans trimmed with postcode=postcode %}
```

Comment:
I think `postcode=postcode` is redundant?

Reply:
Ah so vars can be read in the blocks, just not vars read from the an object? I'll remove

Reply:
Double checked, you were right so I have removed

Reply:
Yeah, it's just complex vars that don't work. Using a filter on `postcode` wouldn't work either